source/_integrations/lamarzocco.markdown

```diff
@@ -67,77 +67,81 @@ Use Bluetooth:
 
 # Available platforms & entities
 
+{% icon "material-symbols:cloud-outline" %} = La Marzocco Cloud
+<iconify-icon inline title="Bluetooth" icon="material-symbols:bluetooth"></iconify-icon> = Bluetooth
+{% icon "material-symbols:wifi" %} = Local network
```

Comment:
Maybe show this as a list?

I like that you use icons to make it clear to users what is supported ❤️

Reply:
I thought about that, but didn't want to use too much estate for the legend. I think it's pretty neat as well, just waiting for another PR, so the icons have proper titles e.g. for screen readers.